### PR TITLE
feat: hide password #181

### PR DESCRIPTION
**Pull request by FabioSM46**
Add checkbox to hide-show password in login sign up page

### CODE REVIEW

File: package-lock.json
Comment (hereisadi) [inline package-lock.json:1]: pnpm-lock.yaml file is already created, so need to create another package-lock.json. Most probably, you used `npm i` to install the node_modules; rather use `pnpm i` to install the dependencies and delete the package-lock.json

**Review by hereisadi — CHANGES_REQUESTED** [review]

**Comment by hereisadi** [comment]
#181

**Comment by hereisadi** [comment]
Thanks for the follow-up. There are some conflicts involved. Can you please, take a pull from `vite_newdev` branch?

**Comment by FabioSM46** [comment]
@hereisadi conflicts are solved.

**Review by hereisadi — APPROVED** [review]

**Comment by hereisadi** [comment]
Thanks, merged!

**Pull request merged**